caffe2_builders/v140/auto.bat

```diff
@@ -0,0 +1,105 @@
+rem Remove to original folder after script is finished
+set ORIGINAL_DIR=%cd%
+
+rem Build folders are 
+rem %CAFFE2_ROOT%\build\Debug for Debug and
+rem %CAFFE2_ROOT%\build\Release for Release
+set CAFFE2_ROOT=%~dp0%\pytorch
+
+rem Cloning repository if it doesn't exist
+if not exist pytorch (
+    set PYTORCH_BRANCH=windows-shared-build
+    set PYTORCH_REPO=ArutyunovG
+    call  %~dp0%..\..\internal\clone.bat
```

Comment:
According to CI, we will have to change dir first. Otherwise, the cloned repo will be somewhere else.